designer/server/src/main/scala/pl/touk/nussknacker/ui/api/DefinitionResources.scala

```diff
@@ -37,6 +40,12 @@ class DefinitionResources(modelDataProvider: ProcessingTypeDataProvider[ModelDat
           modelDataProvider.mapValues(_.processDefinition.services.mapValuesNow(UIProcessObjectsFactory.createUIObjectDefinition(_, processCategoryService))).all
         }
       }
+    } ~ path("processDefinitionData" / "categoriesWithProcessingType") {
```

Comment:
One thing more. This is not processDefinitionData, but just settings / config?

Reply:
I've moved it next to the config endpoint.

Reply:
Ofc, It will stay here forever :)